### DISCUSSION TITLE
Documentation feedback

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#1144] [Feature feedback]
Let's use this space to provide feedback for anything documentation-related.

**Comment by sedubois** [comment]
I just discovered that custom [actions](https://docs.avohq.io/3.0/actions) can be added to resources also in the free version of Avo. For a long time I thought this was an Enterprise-only feature because I was making a search on "custom" in the documentation which led me to the [Customizable controls](https://docs.avohq.io/3.0/customizable-controls.html) documentation. I now know that there is an entire section in the left-side menu called Customize Avo, but the "Customizable controls" is above that in the navigation so that is where I ended up.

1. It would be very helpful to link from one documentation page to the other so that people are aware of both features, how they relate to each other, and the fact that they are in different plans. 

2. It would help to clarify the wording in the documentation (and pricing page) between actions, controls, and "customizability". For instance, the customizable controls documentation page itself refers to "actions" (e.g. its section "Conditionally hiding/showing actions"). Also, actions (free feature) are a way to run custom code, so in that light the "customizable controls" naming is confusing to me. And also clarifying the navigation menu if possible, as mentioned above.

3. It would also be helpful to link from the Actions doc page to the video demonstrating how to implement such actions (the [video](https://www.youtube.com/watch?v=BK47E7TMXn0) shown on avohq.io, at the 13 min mark. Also adding an equivalent link if there's a video somewhere demonstrating the enterprise "customizable controls" feature.

**Reply by adrianthedev** [reply]
Top notch feedback!
I added it to an issue so we can track it as a feature and declare it "done".

**Reply by adrianthedev** [reply]
To address a few things:

 - the customizable controls is and Advanced, not an Enterprise feature
 - each feature (most features) have on [the top of the page](https://docs.avohq.io/3.0/customizable-controls.html#conditionally-hiding-showing-actions:~:text=European%20Ruby%20Conference-,License%3A%20Advanced,-Customizable%20controls) the license required.
 - some features have that [in-line](https://docs.avohq.io/3.0/search.html#global-search:~:text=the%20Index%20view.-,Global%20search,License%3A%20Pro,-Avo%20also%20has)
 - > It would help to clarify the wording in the documentation (and pricing page) between actions, controls, and "customizability".
 
I thought we did. Actions are a standalone feature, but the resource controls implement the actions feature, so that's why we talk about hiding/showing actions in the controls feature. We're [open to contributions on the docs](https://github.com/avo-hq/avodocs). Please help us!
- added the video link for actions
- we'll do more videos this summer


![CleanShot 2024-03-02 at 14 22 55@2x](https://github.com/avo-hq/avo/assets/1334409/dba28601-ccd6-4af9-8e4d-63ce80b0d416)

Thanks for the feedback!
